compatibility-verifier/compCheck.sh

```diff
@@ -43,7 +43,7 @@
 # get usage of the script
 function usage() {
   command=$1
-  echo "Usage: $command [workingDir]"
+  echo "Usage: $command [workingDir] testSuiteDir"
```

Comment:
Looks like `workingDir` is not optional, as per line 153 and 160. So, remove the square brackets
```suggestion
  echo "Usage: $command workingDir testSuiteDir"
```

Reply:
done!